### ISSUE TITLE
There is an issue with fire support

### ISSUE THREAD

**Issue by tuxfuh** [hyper1on/Empire-1_5-Continued-#7]
When I tried to use fire support, all the shells landed in the lower left corner of the map grid, and I'm not sure what caused it

Exception in RimWorld.Planet.WorldComponentUtility.WorldComponentTick: System.NullReferenceException: Object reference not set to an instance of an object
[Ref 9CBBD3A0]
at FactionColonies.MilitaryFireSupport.DoCombatExtendedLaunch (Verse.IntVec3 spawnCenter, Verse.ThingDef def) [0x0021e] in <ab76a4eaa51b4916839a337f94f79d9b>:0 
at FactionColonies.MilitaryFireSupport.Process () [0x0003b] in <ab76a4eaa51b4916839a337f94f79d9b>:0 
at FactionColonies.FactionFC+<>c.<FireSupportTick>b__101_1 (FactionColonies.MilitaryFireSupport support) [0x00000] in <ab76a4eaa51b4916839a337f94f79d9b>:0 
at System.Collections.Generic.List`1[T].ForEach (System.Action`1[T] action) [0x00024] in <eae584ce26bc40229c1b1aa476bfa589>:0 
at FactionColonies.FactionFC.FireSupportTick () [0x0005c] in <ab76a4eaa51b4916839a337f94f79d9b>:0 
at FactionColonies.FactionFC.WorldComponentTick () [0x000d5] in <ab76a4eaa51b4916839a337f94f79d9b>:0 
at RimWorld.Planet.WorldComponentUtility.WorldComponentTick (RimWorld.Planet.World world) [0x00016] in <2a40c3593b334f29ac3cb3d32d652351>:0 
- TRANSPILER net.pardeike.rimworld.lib.harmony: IEnumerable`1 VisualExceptions.ExceptionsAndActivatorHandler:Transpiler(IEnumerable`1 instructions, MethodBase original)

**Comment by Taorer** [comment]
Yeah I had the same problem, after some testing I realised that It happens when Combat Extended and this mod is also in the mod list, so Its purely a CE compatibility problem.

**Comment by Mitril** [comment]
So CE, huh, suprised it causes the problem when it's just the fire support placement
Is there a solution for that?
Maybe changing mod order with empire and CE?
